lib/Net/LibNFS.pm

```diff
@@ -152,7 +152,7 @@ at once. Clobbers (and is clobbered by) C<uid> and C<gid>.
 
 =item * C<pagecache>, C<pagecache_ttl>, C<readahead>
 
-=item * C<readmax>, C<writemax>
+=item * C<readmax>, C<writemax>, C<readdir_buffer>
```

Comment:
I wonder if `readdir_buffer_size` would be easier to grok. At first blush I’m not sure if this is a boolean or what.

Reply:
I used the shorter form upstream because it's faster to type in the URLs. I can change it here, but then it would differ from upstream. In any case this needs more docs to explain that either a scalar or array ref can be used.

Reply:
> but then it would differ from upstream

How so? Upstream has `nfs_set_readdir_max_buffer_size`?

Reply:
Yes, that's the function name. However the example tools like `nfs-ls` take a URL, and there it's the shorter form:
https://github.com/sahlberg/libnfs/blob/4f7f645cd8d9a7c6175b6ce986c92e311a809bda/lib/libnfs.c#L338-L347

Because well, `nfs://some.host/the/export/path?readdir_max_buffer_size=65536` felt too clunky to type. There was a precedent for `nfs_set_interface()` where the option is named `if`.

Anyway, up to you, just let me know how the option should be named in the Perl module.